lib/workers/push_change_handler.rb

```diff
@@ -26,13 +26,15 @@ def submit_push_for_processing!(push)
   private
 
   def set_status_for_push!(push) # rubocop:disable Naming/AccessorMethodName
-    api = Github::Api::Status.new(Rails.application.secrets.github_user_name,
-                                  Rails.application.secrets.github_password)
-    api.set_status(push.branch.repository.name,
-                   push.head_commit.sha,
-                   CONTEXT_NAME,
-                   push.status,
-                   STATE_DESCRIPTIONS[push.status.to_sym],
-                   url_for(controller: '/jira/status/push', action: :edit, id: push.head_commit.sha))
+    if push.service.name == "web"
```

Comment:
push has a `service_name` delegate, so you can use `push.service_name`

Reply:
Ah, nice! Done here: 38f87d3f7ddfd27991fc0ec51809746889823b25